### DISCUSSION TITLE
Room Card

### DISCUSSION THREAD

**Discussion by joe-winter** [UI-Lovelace-Minimalist/UI#1525] [Q&A]
Is it possible to have the template that controls the colour of the room card apply based on a conditon. For example if presence is detected, the room card becomes green.

**Comment by guger** [comment]
You can set the entity_id to that of the presence detector and add a color template, `blue_on`, for example, to the template configuration of the card.

See: https://ui-lovelace-minimalist.github.io/UI/usage/cards/card_room/#room-entity
